.github/workflows/pr-link-check.yml

```diff
@@ -0,0 +1,63 @@
+name: Check Links In Pull Requests
+permissions: {}
```

Comment:
also format this workflow the same as others, put permissions lower on its own block etc.